src/next.md

```diff
@@ -7,6 +7,190 @@
+
+A few things to keep in mind when filing a new bug:
+
+* Before filing a bug, verify that there isn't already a bug filed for this issue.
+* Make a reasonable attempt to narrow down which build or release the failure first appeared in.
+* Add relevant labels like `intermittent`, `regression`, `noreg-self` etc.
+* Set affects version to the JDK version(s) where the failure was seen.
+  * If the failure is found in an update train of the JDK (e.g. 11.0.x), please make an effort to see if the bug is also present in [mainline](https://hg.openjdk.java.net/jdk/jdk/).
+* In the description, always include (if possible):
+  * full name of the failing tests
+  * error messages
+  * assert messages
+  * stack trace
+  * command line information
+  * relevant information from the logs
+* If the failure is not reproducible with an existing OpenJDK test, attach a reproducer if possible.
+* Only set CPU and/or OS fields if the bug **ONLY** happens on that particular platform.
+* Always file separate bugs for different issues.
+  * If two crashes looks related but not similar enough to for sure be the same, it is easier to close a bug as a duplicate than it is to extract the relevant info from a bug to create a new one later.
+
+## Excluding a Test
+
+Sometimes tests break. It could be e.g. due to bugs in the test itself, due to changed functionality in the code that the test is testing, or changes in the environment where the test is executed. While working on a fix, it can be useful to stop the test from being executed in everyone else's testing to reduce noise, especially if the test is expected to fail for more than a day. There are two ways to stop a test from being run in standard test runs: ProblemListing and using the `@ignore` keyword. Removing tests isn't the standard way to remove a failure. A failing test is often a regression and should ideally be handled with high urgency.
+
+I'll say it right away so that it's not forgotten at the end: Remember to remove the JBS id from the ProblemList or the test when the bug is closed. This is especially easy to forget if a bug is closed as a duplicate or 'Will Not Fix'.
+
```

Comment:
I think @iignatev pushed a jcheck update that will fail if the bug in the commit matches one in the problem list and it isn't being removed in the commit. So maybe we can add a note about that.

Reply:
Yes, good point! I have added a sentence about that.